brainscore_language/plugins/sg_tse/benchmark.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from brainscore_core.benchmarks import BenchmarkBase
+from brainscore_language import load_metric, benchmarks
+from brainscore_language.plugins.sg_tse.data import user_def_params
+from brainscore_language.models.huggingface import HuggingfaceSubject
+from brainscore_language.artificial_subject import ArtificialSubject
+from brainscore_language.plugins.sg_tse import compute_surprisals, evaluate
+
+logger = logging.getLogger(__name__)
+
+BIBTEX = """MIT Psycholinguistics lab; SyntaxGym, LM-Zoo; ,
```

Comment:
that's a weird BIBTEX key, is that what they used?

Reply:
Removed.  No it wasn't from them.